src/main/resources/application.yaml

```diff
@@ -1,7 +1,7 @@
 kkv:
+  namespace: ${NAMESPACE}
```

Comment:
By convention we use POD_NAMESPACE when depending on the [downward API](https://kubernetes.io/docs/concepts/workloads/pods/downward-api/) for envs.

Alternatively I think we should use `WATCH_NAMESPACE` or something like `TARGET_SERVICE_NAMESPACE` if we think that watching a different namespace is a reasonable use case.

Reply:
I'll move `namespace` and `resync-period` to `kkv.target.service` (next to service name and port), since they are directly related to this. 

I think that `TARGET_NAMESPACE`, or even more descriptive `TARGET_SERVICE_NAMESPACE` is a good idea. Even though we always run kkv in the same namespace as the service, I really don't think there are any technical restrictions to allowing other namespaces as well.